src/clients/RTorrent.ts

```diff
@@ -451,6 +461,60 @@ export default class RTorrent implements TorrentClient {
 		await this.methodCallP<void>("d.check_hash", [infoHash]);
 	}
 
+	async resumeInjection(
+		infoHash: string,
+		decision: DecisionAnyMatch,
+		options: { checkOnce: boolean },
+	): Promise<void> {
+		let sleepTime = resumeSleepTime;
+		const maxRemainingBytes = getMaxRemainingBytes(decision);
+		const stopTime = getResumeStopTime();
+		let stop = false;
+		while (Date.now() < stopTime) {
+			if (options.checkOnce) {
+				if (stop) return;
+				stop = true;
+			}
+			await wait(sleepTime);
+			const torrentInfoRes = await this.checkOriginalTorrent(
+				infoHash,
+				false,
+			);
+			if (torrentInfoRes.isErr()) {
+				sleepTime = resumeErrSleepTime; // Dropping connections or restart
+				continue;
+			}
+			const torrentInfo = torrentInfoRes.unwrap();
+			if (torrentInfo.hashing) {
+				continue;
+			}
+			const torrentLog = `${torrentInfo.name} [${sanitizeInfoHash(infoHash)}]`;
+			if (torrentInfo.isActive) {
+				logger.warn({
+					label: Label.RTORRENT,
+					message: `Will not resume torrent ${torrentLog}: active`,
+				});
+				return;
+			}
+			if (torrentInfo.bytesLeft > maxRemainingBytes) {
+				logger.warn({
+					label: Label.RTORRENT,
+					message: `Will not resume torrent ${torrentLog}: ${humanReadableSize(torrentInfo.bytesLeft, { binary: true })} remaining > ${humanReadableSize(maxRemainingBytes, { binary: true })}`,
+				});
+				return;
+			}
+			logger.info({
```

Comment:
maybe verbose for these? also fine to wait and see what people say

Reply:
In the case of partial, I think it's worth logging in info. Especially since this hangs non-daemon mode.